packages/eslint-plugin/docs/rules/no-extra-non-null-assertion.md

```diff
@@ -50,3 +50,7 @@ function foo(bar?: { n: number }) {
   return bar?.n;
 }
 ```
+
+## When Not To Use It
+
+If you don't mind excess `!`s in your code, you can avoid this rule.
```

Comment:
same as above

"Use this rule if you want to remove extra non-null-assertions"
"Don't use this rule if you are okay with extra non-null-assertions"

Reply:
...yeah I truly cannot think of a reason not to use this rule 😄